pennylane_qiskit/qiskit_device.py

```diff
@@ -74,6 +74,13 @@
     "Adjoint(S)": lib.SdgGate,
     "Adjoint(T)": lib.TdgGate,
     "Adjoint(SX)": lib.SXdgGate,
+    "CY": lib.CYGate,
+    "CH": lib.CHGate,
+    "CPhase": lib.CPhaseGate,
+    "CCZ": lib.CCZGate,
+    "ECR": lib.ECRGate,
+    "Barrier": lib.Barrier,
+    "Adjoint(GlobalPhase)": lib.GlobalPhaseGate,
```

Comment:
Is there tests for these?

Reply:
My understanding was that these gets checked with the `integration-tests`, which runs it with  the `devices/tests/test_gates.py` from the PennyLane.

Reply:
Might be best to follow up with an additional plugin-specific test too anyway.

Reply:
Added it!